README.md

```diff
@@ -25,10 +25,23 @@ It’s a Rails app that:
 
    ```sh
    $ bundle install
+   ```
+
+5. Create development database 
+
+   ```sh
+   $ createdb webpage-versions-db_development
```

Comment:
It would be a little better to see the config files modified to be `web-monitoring-db_development`. This name is a holdover from the previous name of this project and I should have changed it when the project was renamed :\